SCClassLibrary/Platform/osx/OSXPlatform.sc

```diff
@@ -76,4 +76,11 @@ OSXPlatform : UnixPlatform {
 		file.putString(string);
 		file.close;
 	}
+
+	defaultTempDir {
+		var dir;
+		dir = Platform.userAppSupportDir +/+ "tmp";
```

Comment:
@geoffroymontel Now I believe that the trailing `/` is missing here

Reply:
@gusano I've tested the code and it seems to work without the trailing slash. Is it ok for you ?

Reply:
why would the trailing slash be necessary? @gusano

Reply:
@brianlheim it seems that `Platform.defaultTempDir()` [already returns a trailing slash by default](https://github.com/supercollider/supercollider/blob/fe0a60e1b25b421b2c8aeffa6b096653e2176304/SCClassLibrary/Platform/Platform.sc#L223-L224) so it might be better to keep it that way to not break existing code?

@geoffroymontel I'm surprised that it works, I cannot test it now but my guess is that the file might end being
`"/path-to-app-support-dir/tmpasScore-Help.aif"`

Reply:
ok, fair, did not know that. everyone should be using +/+ to concatenate paths though precisely to not get burned by this kind of detail. i believe other platform directory methods return paths without the trailing slash.